experimental/emulated_vault/README

```diff
@@ -0,0 +1,23 @@
+The emulated_vault module supplies a HTTP server mimicking RIAK behavior for the usage as traffic-control vault.
+The server may use different type of persistent storage (e.g. file-system), using the proper adaptor.
+Basic requirements: Centos ver >= 7; Python
+
+In order to deploy the code on a server please:
+1. Copy the module files to the server's root
+2. Add the certificate and key to you favorite path
+3. Adjust opt/emulated_vault/conf/cfg.json - pointing at your certificate and key
+4. chkconfig the service
```

Comment:
`chkconfig` is retired in CentOS version >= 7 as is required. I think you want to say something about a systemd command here?

Reply:
systemctl enable ;)
Thanks for catching it